### PR TITLE
Rephraze "There is no member in this circle"

### PR DESCRIPTION
**Pull request by aryatalathi**
Changes "There is no member in this circle" to "You currently have no access to the memberlist".

Fixes https://github.com/nextcloud/contacts/issues/2284

### CODE REVIEW

**Review by ChristophWurst — CHANGES_REQUESTED** [review]
Please undo/revert the changes in `l10n/`. The translations are handled by Transifex via an automation. Those files are not touched by us developers :)

**Comment by aryatalathi** [comment]
> Please undo/revert the changes in `l10n/`. The translations are handled by Transifex via an automation. Those files are not touched by us developers :)

Okay. I have reverted the changes in `l 10n/`. Kindly check. Thank you!

File: src/components/AppNavigation/ContactsSettings.vue [hunk 1]
```diff
@@ -170,8 +170,7 @@ export default {
 .settings-new-addressbook >>> .icon-confirm {
 	min-height: 44px;
 	height: 44px;
-	border-color: var(--color-border-dark) !important;
-	border-left: none;
+	border-color: black !important;
```
[the remark this answers is not in the document]
Comment (aryatalathi) [reply src/components/AppNavigation/ContactsSettings.vue:173]: Requesting to review and merge. 
The required changes are made.
Thank you!

**Comment by aryatalathi** [comment]
Respected sir,
I have made only required changes and have reverted the unnecessary
commits. Kindly check so the PR can get merged. Thank you.

**Comment by ChristophWurst** [comment]
Please see https://github.com/nextcloud/contacts/pull/3106/files. The requested revert has not been done.

**Comment by aryatalathi** [comment]
> Please see https://github.com/nextcloud/contacts/pull/3106/files. The requested revert has not been done.

I have tried to revert it, but its not reflecting here. May I push the required changes on new branch and create a new pull request?

**Comment by ChristophWurst** [comment]
Sure

**Comment by aryatalathi** [comment]
Kindly check #3133  <https://github.com/nextcloud/contacts/pull/3133>.

**Pull request closed without merging**